agent-bootstrap/src/main/java/io/opentelemetry/auto/instrumentation/api/MoreAttributes.java

```diff
@@ -19,6 +19,7 @@
 public class MoreAttributes {
   public static final String HTTP_QUERY = "http.query.string";
   public static final String HTTP_FRAGMENT = "http.fragment.string";
+  public static final String HTTP_RESPONSE_CONTENT_LENGTH = "http.response_content_length";
```

Comment:
If I recall correctly this was already released in otel 0.6.0, is it not part of `SemanticAttributes`?

Reply:
I did a project wide search for the string before adding it. Is that defined elsewhere?

Reply:
Yeah it's in the SDK repo

https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java